frontend/src/SettingsView/BillingView/BillingView.js

```diff
@@ -32,6 +26,7 @@ const BillingView = () => {
 					token={token =>
 						setPremium({
 							variables: {
+								//need to un hard code this later
```

Comment:
ToDo Tree is a great VScode addon to help keep track of things like this :)
https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree